.gitpod.yml

```diff
@@ -1,4 +1,4 @@
-image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:aledbf-trigger.4
+image: eu.gcr.io/gitpod-core-dev/dev/dev-environment:aledbf-devtest.5
```

Comment:
What do you think about simplifying our dev setup like so?

```suggestion
image:
  file: dev/image/Dockerfile
```

Reply:
done